easybuild/tools/filetools.py

```diff
@@ -655,7 +655,7 @@ def guess_patch_level(patched_files, parent_dir):
     return patch_level
 
 
-def apply_patch(patch_file, dest, fn=None, copy=False, level=None):
+def apply_patch(patch_file, dest, fn=None, copy=False, level=None, log_ok=True, from_pr=None, obliged=False):
```

Comment:
also, I don't think we should make `apply_patch` aware whether is was called in the context of `--from-pr` or not, just catch a possible patching failure where you call `apply_patch` and react to it?